pom.xml

```diff
@@ -33,6 +33,7 @@
 
     <version.certified-community>2017.10.1</version.certified-community>
     <version.wildfly.swarm>${project.version}</version.wildfly.swarm>
+    <version.wildfly.config-api>1.3.1</version.wildfly.config-api>
```

Comment:
was it failing with version in build-parent?

Reply:
Actually that's a good point, the inheritance hierarchy is a bit more complex than I thought. I'll check again.

Reply:
Thanks @michalszynkiewicz, indeed it worked fine even without moving the property. Nice :-)